plume-cli/src/instance.rs

```diff
@@ -0,0 +1,63 @@
+use clap::{Arg, ArgMatches, App, SubCommand};
+use diesel::PgConnection;
```

Comment:
shouldn't this be database agnostic now?

Reply:
Oops, forgot it! :grin:

Reply:
Fixed